aeron-driver/src/main/c/aeron_network_publication.c

```diff
@@ -1224,6 +1273,16 @@ void aeron_network_publication_on_time_event(
             break;
         }
 
+        case AERON_NETWORK_PUBLICATION_STATE_REVOKED:
+        {
+            aeron_driver_conductor_publication_transition_to_revoked(conductor, &publication->conductor_fields.managed_resource);
+            aeron_driver_conductor_cleanup_spies(conductor, publication);
+
+            publication->conductor_fields.time_of_last_activity_ns = now_ns;
+            publication->conductor_fields.state = AERON_NETWORK_PUBLICATION_STATE_LINGER;
```

Comment:
I don't like that the state transitions are different for IPC vs. Network publications.